torch/__init__.py

```diff
@@ -69,6 +69,7 @@ def _running_with_deploy():
     pfiles_path = os.getenv('ProgramFiles', 'C:\\Program Files')
     py_dll_path = os.path.join(sys.exec_prefix, 'Library', 'bin')
     th_dll_path = os.path.join(os.path.dirname(__file__), 'lib')
+    appdata_dll_path = os.path.join(os.getenv('APPDATA'), 'Python', 'Library', 'bin')
```

Comment:
This should be one of the sysconfig flavors, let me try to find out which one...